how-to/get-started/quick-start-with-tidb.md

```diff
@@ -0,0 +1,320 @@
+
+    ```shell
+    mysql --host 127.0.0.1 --port 4000 -u root
+    ```
+
+6. Access the Prometheus dashboard of TiDB at <http://127.0.0.1:9090>.
+
+7. Access the TiDB Dashboard at <http://127.0.0.1:2379/dashboard>. The default username is `root`, with an empty password.
+
+8. After the test, you can clean the cluster by taking the following steps:
+
+    1. Stop the process by pressing `ctrl-c`.
+
+    2. Run the following command:
+
+        {{< copyable "shell-regular" >}}
+
+        ```shell
+        tiup clean --all
+        ```
+
+## Deploy a quasi-production test environment on a single machine using TiUP cluster
+
+- Scenario: Experience a smallest TiDB cluster with the complete topology and simulate the production deployment steps on a single Linux server.
+- Time required: 10 minutes
+
+This section describes how to deploy a TiDB cluster using a YAML file of the smallest topology in TiUP.
+
+### Prepare
+
+Prepare a deployment machine that meets the following requirements:
+
+- CentOS 7.3 or a later version is installed
+- The Linux OS has access to the Internet, which can be used to download TiDB and related software installation packages
+
+The smallest TiDB topology is as follows:
+
+| Instance | Number | IP | Configuration |
+|:-- | :-- | :-- | :-- |
+| TiKV | 3 | 10.0.1.1 <br> 10.0.1.1 <br> 10.0.1.1 | Avoid conflict between the port and the directory |
+| TiDB | 1 | 10.0.1.1 | The default port <br> Global directory configuration |
+| PD | 1 | 10.0.1.1 | The default port <br> Global directory configuration |
+| TiFlash | 1 | 10.0.1.1 | The default port <br> Global directory configuration |
+| Monitor | 1 | 10.0.1.1 | The default port <br> Global directory configuration |
+
+Other requirements for the deployment machine:
+
+- Know the root user password
+- Stop the firewall service of the deployment machine, or open the port needed by the TiDB cluster nodes
+- Currently, TiUP only supports deploying TiDB on the x86_64 (AMD64) architecture (the ARM architecture will be supported in TiDB 4.0 GA):
+
+    - It is recommended to use CentOS 7.3 or later versions on AMD64
+    - It is recommended to use CentOS 7.6 1810 on ARM
+
+### Deploy
+
+> **Note:**
+>
+> You can log in to the host as any ordinary user or the root user of the Linux OS. The following steps use the root user as an example.
+
+1. Download and install TiUP:
+
+    {{< copyable "shell-regular" >}}
+
+    ```shell
+    curl --proto '=https' --tlsv1.2 -sSf https://tiup-mirrors.pingcap.com/install.sh | sh
+    ```
+
+2. Install the cluster component of TiUP:
+
+    {{< copyable "shell-regular" >}}
+
+    ```shell
+    tiup cluster
+    ```
+
+3. If the TiUP cluster is already installed on the machine, update the software version:
+
+    {{< copyable "shell-regular" >}}
+
+    ```shell
+    tiup update cluster
+    ```
+
+4. Use the root user access to increase the connection limit of the `sshd` service. This is because TiUP needs to simulate deployment on multiple machines.
```

Comment:
```suggestion
4. Use the `root` user privilege to increase the connection limit of the `sshd` service. This is because TiUP needs to simulate deployment on multiple machines.
```